### PR TITLE
feat(atom/popover): new component Popover

### PR DESCRIPTION
**Pull request by davidmartin84**
Atom Popover - [SUIC-201](https://jira.scmspain.com/browse/SUIC-201)

### CODE REVIEW

File: components/atom/popover/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+$c-atom-popover: $c-black !default;
+$bdc-atom-popover: $c-gray-light-3 !default;
+$bgc-atom-popover: $c-white !default;
+$bdrs-atom-popover: $bdrs-m !default;
+$bdw-atom-popover: 1px !default;
+$pt-atom-popover: $p-m !default;
+$pb-atom-popover: $p-m !default;
+$pl-atom-popover: $p-m !default;
+$pr-atom-popover: $p-m !default;
+$m-atom-popover: 0 !default;
```
Comment (turolopezsanabria) [inline components/atom/popover/src/index.scss:14]: Without seeing the Demo: is it possible that margin and paddings are inverted?
I mean Padding (space within the popover) should be 0, and the Margins (space to the parent element) $m-m?

**Comment by nucliweb** [comment]
## Proposal

### Popover no arrow, no close icon

Popovers no arrrow no close-icon

### Without padding

No padding

File: components/atom/popover/src/index.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+function AtomPopover({
+  children,
+  content = () => {},
+  id,
+  onClose = () => {},
+  placement = PLACEMENTS.BOTTOM
+}) {
+  targetRef = useRef()
+  const [showPopover, setShowPopover] = useState(false)
+  Popover = React.lazy(() => import('reactstrap/lib/Popover'))
```
Comment (midudev) [inline components/atom/popover/src/index.js]: This should be performed outside the component.

File: components/atom/popover/src/index.scss (hunk 1 above)
Comment (nucliweb) [reply components/atom/popover/src/index.scss:14]: @turolopezsanabria you can view the demo [here](https://sui-components-pr-891-qlpvpzyaur.now.sh/workbench/atom/popover/demo)

File: components/atom/popover/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+```sh
+$ npm install @s-ui/react-atom-popover --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import AtomPopover from '@s-ui/react-atom-popover'
```
Comment (nucliweb) [inline components/atom/popover/README.md]: You need import `atomPopoverPositions` to use it below.

```js
import AtomPopover, { atomPopoverPositions } from '@s-ui/react-atom-popover'
```

**Review by jcger — APPROVED** [review]

**Review by nucliweb — APPROVED** [review]

**Review by jordevo — APPROVED** [review]

**Review by quinwacca — APPROVED** [review]

**Review by kikoruiz — APPROVED** [review]

**Review by carlosvillu — APPROVED** [review]

**Pull request merged**